aries-site/src/pages/foundation/color.js

```diff
@@ -22,15 +23,11 @@ const Color = () => (
       </MainDescription>
       <Box gap="medium" pad={{ vertical: 'medium' }}>
         <Button
-          label={
-            <Text
-              color={{ dark: 'text-strong', light: 'text-strong' }}
-              weight={500}
-            >
-              Download Color Swatches Pack
-            </Text>
-          }
+          label="Download Color Swatches Pack"
```

Comment:
Getting an accessibility error for this color combination of white text on green background (see my other comment for more details)

Reply:
Yes, we need to surface this again.